xml/schema/types/oblocks-2-9-6.xsd

```diff
@@ -43,9 +43,9 @@
         <xs:element name="oblock" minOccurs="0" maxOccurs="unbounded">
           <xs:complexType>
             <xs:sequence>
```

Comment:
A clarification: XSD 1.0 requires maxOccurs to be 0 or 1 within a xs:all element, while XSD 1.1 allows maxOccurs to be any value including unbounded.